versions/3.0.4.md

```diff
@@ -171,6 +171,8 @@ Two formats, `binary` and `byte`, describe different ways to work with binary da
 * `binary` is used where unencoded binary data is allowed, such as when sending a binary payload as an HTTP message body, or as part of a `multipart/*` payload that allows binary parts
 * `byte` is used where binary data is embedded in a text-only format such as `application/json` or `application/x-www-form-urlencoded`
 
+The `maxLength` keyword can be used to set an upper bound on the length of a binary payload; otherwise, a binary payload SHOULD be interpreted as a stream of indeterminate length.
```

Comment:
```suggestion
The `maxLength` keyword can be used to set an upper bound on the length (in octets) of a binary payload; otherwise, a binary payload SHOULD be interpreted as a stream of indeterminate length.
```